_i18n/en.yml

```diff
@@ -30,7 +30,7 @@ titles:
   workgroups: Workgroups
   hangouts: Hangouts
   sponsorships: Sponsorships
-  merchants: Merchants & Services
+  merchants: Merchants & Exchanges
```

Comment:
I would also change the menu item (line 55 of this file) to `Merchants & Exchanges`

Reply:
That would make it too long to fit on our menu bars i'm afraid.

Reply:
It's not really a problem because multiple languages already have menu items using two lines:
![image](https://user-images.githubusercontent.com/45968869/131464930-a8d86647-f43d-44b0-8f90-551a187b664a.png)

Reply:
Some languages do have too long menu items and in some sections it's a problem. Personally here i don't care. This is how it 
would look like:
![MENU](https://user-images.githubusercontent.com/28106476/131466251-4adeb890-d4d9-4271-999a-e2867c837014.png)

It breaks the format at least on my screen, by being the only entry on two line (in all dropdown menus), but if people are ok with that i'll just change it.

Reply:
Yes, this is the only menu item in English version that has a line break, but I don't see this as a problem since all other languages have at least one menu item with a line break.

Reply:
Agreed. Clarity is more important than consistency in this case.